src/org/benf/cfr/reader/util/output/StreamDumper.java

```diff
@@ -174,12 +177,16 @@ private void processPendingCR() {
 
     @Override
     public void indent(int diff) {
-        context.indent += diff;
+        if (context.inBlockComment == BlockCommentState.Not) {
+            context.indent += diff;
+        } else {
+            context.blockCommentIndent += diff;
+        }
     }
 
     @Override
     public int getIndentLevel() {
-        return context.indent;
+        return context.indent + context.blockCommentIndent;
```

Comment:
You could probably also omit this overridden method because `AbstractDumper.getIndentLevel()` does exactly the same.

Reply:
Good idea